openapi/components/schemas/CreditMemos/CreditMemoProperties.yaml

```
@@ -0,0 +1,176 @@
+                If the `amount` value is not supplied, the lesser of the following two values is used:
+                - The unused amount of the credit memo.
+                - The transaction amount.
+              type: number
+              format: double
+            currency:
+              readOnly: true
+              allOf:
+                - $ref: ../CurrencyCode.yaml
+            createdTime:
+              description: Credit memo allocation time.
+              readOnly: true
+              allOf:
+                - $ref: ../ServerTimestamp.yaml
+            updatedTime:
+              $ref: ../UpdatedTime.yaml
+items:
+  type: array
+  items:
+    type: object
+    required:
+      - unitPrice
+      - quantity
+    properties:
+      id:
+        description: Credit memo item ID.
+        readOnly: true
+        allOf:
+          - $ref: ../ResourceId.yaml
+      invoiceItemId:
+        description: An invoice item's ID that the credit item is referenced to.
+        nullable: true
+        allOf:
+          - $ref: ../ResourceId.yaml
+      description:
+        description: Credit memo item's description.
+        type: string
+      unitPrice:
+        description: Credit memo item's price.
+        type: number
+        format: double
+      quantity:
+        description: Credit memo item's quantity.
+        type: integer
+      price:
+        description: Credit memo item's total price.
+        type: number
+        format: double
+        readOnly: true
+      productId:
+        description: The related product's ID.
+        nullable: true
+        allOf:
+          - $ref: ../ResourceId.yaml
+      planId:
+        description: The related plan's ID.
+        nullable: true
+        allOf:
+          - $ref: ../ResourceId.yaml
+status:
+  type: string
+  description: Status of the credit memo.
+  readOnly: true
+  enum:
+    - issued
+    - applied
+    - partially-applied
+    - voided
+  x-enumDescriptions:
+    issued: The credit memo is available with a full amount to be applied.
+    applied: All the amount of the credit memo has been applied, no more credit can be created from it.
+    partially-applied: >-
+      Some amount of the credit memo has been applied,
+      credit of amount `unusedAmount` can be created from it.
+    voided: >-
+      Some credit has or hasn't been created from the credit memo,
+      but the credit memo cannot be used anymore.
+reason:
+  description: Credit memo reason code.
+  type: string
+  enum:
+    - return
+    - product-unsatisfactory
+    - order-change
+    - order-cancellation
+    - chargeback
+    - write-off
+    - waiver
+    - customer-credit
+    - other
+description:
+  type: string
+  x-basic: true
+  description: A public description (visible to customers) clarifying the purpose of the credit memo.
+currency:
+  $ref: ../CurrencyCode.yaml
+shippingAmount:
+  description: The (invoice's) shipping amount to credit.
```

nitpick: suggestion to avoid possessive ('s) & parenthesis 
```suggestion
  description: Shipping amount of an invoice to credit.
```